engines/sci/graphics/helpers.h

```diff
@@ -118,6 +126,90 @@ struct Window : public Port, public Common::Serializable {
 	}
 };
 
+#ifdef ENABLE_SCI32
+/**
+ * Multiplies a rectangle by two ratios with default
+ * rounding. Modifies the rect directly.
+ */
+inline void mul(Common::Rect &rect, const Common::Rational &ratioX, const Common::Rational &ratioY) {
+	rect.left = (rect.left * ratioX).toInt();
+	rect.top = (rect.top * ratioY).toInt();
+	rect.right = (rect.right * ratioX).toInt();
+	rect.bottom = (rect.bottom * ratioY).toInt();
```

Comment:
This scaling might also be affected by my other comment about scaling in the presence of inclusive vs. exclusive bottom-right.